python/client/cortex/client.py

```diff
@@ -260,33 +240,3 @@ def stop_job(self, api_name: str, job_id: str, keep_cache: bool = False):
         ]
 
         run_cli(args)
-
-    @sentry_wrapper
-    def stream_api_logs(
```

Comment:
What about the `stream_api_logs` and `stream_job_logs` from the `tests.py` file? Should we remove those as well, or reimplement these methods for the `--random-pod` case?